LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Model/DataBroker.swift

```diff
@@ -25,6 +25,15 @@ struct DataBrokerScheduleConfig: Codable {
     let confirmOptOutScan: Int
     let maintenanceScan: Int
     let maxAttempts: Int
+
+    var optOutReattempt: Int {
```

Comment:
I'm struggle to understand what this is

Reply:
Added a comment. We need a reasonable number to delay the next opt-out just after the next opt-out scan (confirmOptOutScan), so for now it's set to maintenanceScan.

Reply:
I'm still struggling, what are the units of the number?

Reply:
Same as maintenanceScan and confirmOptOutScan, so the unit is hours.

Reply:
A less confusing implementation will just be 

var reOptOutAttempt: Int { maintenanceScan }

but then if it's later changed and somehow less than confirmationOptOutScan, it'll lead to undesired attempts, hence the assertion.

Reply:
I really think we should rename it to be clearer. I know it matches the others, but they're not great.  E.g. if I see confirmOptOutScan immediately it's baffling that it's an int, and whoever is reading it shouldn't have to guess at the units

Reply:
Good point, I've renamed it to `hoursUntilNextOptOutAttempt`. As for the other properties (maintenanceScan, confirmOptOutScan), let's word them better in a different PR.